model/task_config.go

```diff
@@ -142,7 +142,12 @@ func MakeConfigFromTask(t *task.Task) (*TaskConfig, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "error finding host")
 	}
-	e, err := PopulateExpansions(t, h)
+
+	settings, err := evergreen.GetConfig()
```

Comment:
is it reasonable to pass the settings object4 into this function?

Reply:
The alternative would be to get the oauth token and pass that into PopulateExpansions instead.